### ISSUE TITLE
Fix 'tests/integration/test_remoted/test_agent_communication/test_multi_agent_status' tests in 4.2

### ISSUE THREAD

**Issue by damarisg** [wazuh/wazuh-qa#1576]
### Issue information

| Related issue |
|--|
| Closes https://github.com/wazuh/wazuh-qa/issues/1530|

When it comes to starting with the test fix, there is some information that can help you:
```
Module: Remoted
Support Team: "Core" 
Channel: "devel-core"
Target: Manager   
OS: Linux
```
I add some information obtained while researched the logs that failed.

## Case:

|  Type | Description  |
|--|--|
|Test Path | tests/integration/test_remoted/test_agent_communication/test_multi_agent_status
|Consistent | yes
|Test Execution | 3/3 executions failed
|Cases Fails | 6
|Summary | On `test_protocols_communication` when it called to `check_active_agents`.
|Error message| 'Could not find the remoted key loading log'

In order to finish this issue the following tasks should be fulfilled:
- [ ] Research of fails.
- [ ] Apply Fix 
- [ ] Full Green/ Full Yellow  in `tests/integration/test_remoted/test_agent_communication/test_multi_agent_status` for 3 times. 
- [ ] Documentation of any important change done for these tests or the used tools.

**Comment by mdengra** [comment]
### 2021-07-20

I was unable to reproduce the described error in my testing environment.
Used Wazuh-QA branch: `1516-4.2.0-full-green`
Test results with the default modules enabled in the `ossec.conf`: 

| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[test_multi_agent_status_local_r1.log](https://github.com/wazuh/wazuh-qa/files/6847692/test_multi_agent_status_local_r1.log)| 2021-07-20 | Miguel | :yellow_circle:  
|[test_multi_agent_status_local_r2.log](https://github.com/wazuh/wazuh-qa/files/6847694/test_multi_agent_status_local_r2.log)| 2021-07-20 | Miguel | :yellow_circle:  
|[test_multi_agent_status_local_r3.log](https://github.com/wazuh/wazuh-qa/files/6847695/test_multi_agent_status_local_r3.log)| 2021-07-20 | Miguel | :yellow_circle:

**Comment by juliamagan** [comment]
## Tests results

||CentOS 8 Manager |Reports|
|:--:|:--:|:--:|
|R1|  🟢    | [Tests results](https://github.com/wazuh/wazuh-qa/files/6876628/centos_manager_test_multi_agent_status_r1.zip)| 
|R2|   🟢    | [Tests results](https://github.com/wazuh/wazuh-qa/files/6876629/centos_manager_test_multi_agent_status_r2.zip)
|R3|  🟢    | [Tests results](https://github.com/wazuh/wazuh-qa/files/6876630/centos_manager_test_multi_agent_status_r3.zip) |

**Comment by damarisg** [comment]
This issue will be closed because we can't reproduce it.
Test Results:
| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[YellowMultiAgentStatus.log](https://github.com/wazuh/wazuh-qa/files/6886468/YellowMultiAgentStatus.log)| 2021-07-27 | Seyla | :yellow_circle:  
|[YellowMultiAgentStatus2.log](https://github.com/wazuh/wazuh-qa/files/6886466/YellowMultiAgentStatus2.log)| 2021-07-27 | Seyla| :yellow_circle:  
|[YellowMultiAgentStatus3.log](https://github.com/wazuh/wazuh-qa/files/6886467/YellowMultiAgentStatus3.log)| 2021-07-27 | Seyla| :yellow_circle:

**Comment by damarisg** [comment]
This issue work random, and it affected by https://github.com/wazuh/wazuh-qa/issues/1698. After to fix this problems we can unblock this issue.

**Comment by damarisg** [comment]
This issue is closed because the test was disabled by #1602.
